src/Layout/Navigation/RecentItem.tsx

```diff
@@ -11,26 +11,137 @@
  */
 
 import React from 'react';
-import { Link } from 'react-router-dom';
-import { NavItem } from '@patternfly/react-core';
+import { AlertVariant, Dropdown, DropdownItem, KebabToggle, NavItem } from '@patternfly/react-core';
 import { NavigationRecentItemObject } from '.';
-import isActive from './isActive';
+import getActivity from './isActive';
 import WorkspaceIndicator from '../../components/Workspace/Indicator';
+import WorkspaceActionsProvider from '../../containers/WorkspaceActions';
+import { ActionContextType, WorkspaceActionsConsumer } from '../../containers/WorkspaceActions/context';
+import { WorkspaceAction, WorkspaceStatus } from '../../services/helpers/types';
+import getRandomString from '../../services/helpers/random';
+import { lazyInject } from '../../inversify.config';
+import { AppAlerts } from '../../services/alerts/appAlerts';
+import { History } from 'history';
```

Comment:
I have no idea how it happens, maybe we save error somewhere, but dashboard said me `You're not allowed to start more workspaces`, and started workspace for me.
I did not have running workspaces and used open in verbose mode from kebab menu.
![Screenshot_20210402_095043](https://user-images.githubusercontent.com/5887312/113389635-09e1bb80-9399-11eb-8d7b-8cb912b549d5.png)